lib/spack/spack/cmd/clean.py

```diff
@@ -98,6 +98,8 @@ def clean(parser, args):
     if args.python_cache:
         tty.msg('Removing python cache files')
         for root, dirs, files in os.walk(spack_root):
```

Comment:
I think if you replace this line with:
```python
from spack.paths import lib_path, var_path

for spack_root in [lib_path, var_path]:
    for root, dirs, files in os.walk(spack_root):
```
you should be good to go. Those seem to be the only directories containing anything that gets imported.